public/index.html

```diff
@@ -20,7 +20,7 @@
     <title>SQE</title>
 </head>
 
-<body style="overflow: hidden;">
+<body style="overflow: hidden;background-color: #E5E5E5;">
```

Comment:
Would it be perhaps more visible to put these styles into a dedicated <style> section?  Do whatever you think will be easiest to read (and debug).